client/offchain/src/api.rs

```diff
@@ -96,16 +96,16 @@ impl<Storage: OffchainStorage> OffchainExt for Api<Storage> {
 
 	fn local_storage_set(&mut self, kind: StorageKind, key: &[u8], value: &[u8]) {
 		match kind {
-			StorageKind::PERSISTENT => self.db.set(STORAGE_PREFIX, key, value),
+			StorageKind::PERSISTENT => self.db.set(STORAGE_PREFIX, key, value).unwrap(),
```

Comment:
This is not a test code, so please no `unwraps`. It should be fine to print an error here, since there is not much we can do anyway.

Reply:
done